client/src/shared/components/InputField/index.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable object-shorthand */
+import React from 'react';
+
+export interface InputFieldProps {
+  label?: string;
+  placeholder?: string;
+  type?: string;
+  id?: string;
+  width?: string;
+  height?: string;
+}
+
+const InputField: React.FC<InputFieldProps> = ({
+  label,
+  placeholder,
+  type,
+  id,
+  width,
+  height
+}: InputFieldProps) => {
+  const propStyle = {
+    width: width,
+    height: height
+  };
+
+  return (
+    <div className="mb-4">
+      {label?.toString() !== '' && (
```

Comment:
- simplify 
```suggestion
      {label && (
```

Reply:
okay sir, Ill just add this ESLint rule `/* eslint-disable @typescript-eslint/strict-boolean-expressions */`